projects/igniteui-angular/src/lib/grids/grouping/group-by-area.directive.ts

```diff
@@ -122,7 +122,7 @@ export abstract class IgxGroupByAreaDirective {
         const drag: IgxColumnMovingDragDirective = event.detail.owner;
         if (drag instanceof IgxColumnMovingDragDirective) {
             const column = drag.column;
-            if (this.grid.columns.indexOf(column) < 0) {
+            if (this.grid.columnList.find(c => c === column).index < 0) {

```

Comment:
```suggestion
            if (this.grid.columnList.find(c => c === column)) {

```
The old code was testing if the column is in the grid column collection, not its index